extensions/empty-screenshots/package.json

```diff
@@ -5,6 +5,9 @@
   "description": "Empties your screenshot folder within a configurable time period.",
   "icon": "empty-screenshot-folder.png",
   "author": "chanceaclark",
+  "contributors": [
+    "charlie_deets"
```

Comment:
🍹 
```suggestion
    "robotdestroy"
```

Reply:
From my understanding, this is meant to be my Raycast account name, and not my GitHub name, so this one actually is correct. Thanks for caring! :)

Reply:
That's correct 🙂